blockchain.go

```diff
@@ -449,6 +461,12 @@ func NewBlockchain(opts ...Option) (*Blockchain, error) {
 	if err != nil {
 		return nil, err
 	}
+	if conf.WithContracts != nil && len(conf.WithContracts) > 0 {
```

Comment:
You can just check for len(), because if nil it would just be 0.

Reply:
Ah, good suggestion 🙌

Reply:
Fixed in https://github.com/onflow/flow-emulator/pull/367/commits/b4ab4b80457b775c2c360df6bdc8cfdf98013044